subsystem/infra_env_test.go

```diff
@@ -358,8 +358,10 @@ var _ = Describe("Infra_Env", func() {
 		}
 		Expect(contents).To(ContainSubstring("eth0"))
 	},
-		Entry("ocp versions greater than/ equal to 4.14", staticnetworkconfig.MinimalVersionForNmstatectl),
-		Entry("ocp versions less than 4.14", "4.12"),
+		Entry("ocp versions greater than/ equal to 4.14, x86 arch", staticnetworkconfig.MinimalVersionForNmstatectl, common.X86CPUArchitecture),
+		Entry("ocp versions greater than/ equal to 4.14, arm arch", staticnetworkconfig.MinimalVersionForNmstatectl, common.ARM64CPUArchitecture),
+		Entry("ocp versions less than 4.14, x86 arch", "4.12", common.X86CPUArchitecture),
+		Entry("ocp versions less than 4.14, arm arch", "4.12", common.ARM64CPUArchitecture),
```

Comment:
I am not sure what we are  testing here

Reply:
For OCP versions below 4.14, the static networking flow follows the older process, without using nmpolicy or the nmstate service. In this case, the files under `/etc/assisted/network/host0` will be the nmconnection files, not nmpolicy. This behavior is consistent across all architectures.

It might be worth updating the condition in the test  to:
`if ocpVersion < staticnetworkconfig.MinimalVersionForNmstatectl || arch == common.ARM64CPUArchitecture `